sdcm/results_artifacts.html

```diff
@@ -60,6 +60,20 @@ <h3>
         </ul>
     </div>
 
+    {% if last_events %}
```

Comment:
can we move this to results_base.html ? as a block ?
```
# results_base.html
{% block events_summary %}
..everything here..
{% endblock %}
```

and other type would be able to overwrite it if needed (or add more sub blocks for specific things)